src/Modules.h

```diff
@@ -0,0 +1,58 @@
+#ifndef GSW_MODULES_H
+#define GSW_MODULES_H
+
+class GroundStationWindow{
+public:
+    virtual void RenderGUI()=0;
+    virtual GroundStationWindow* GetPtr()=0;
```

Comment:
```suggestion
    virtual GroundStationWindow* GetPtr() = 0;
```
Not sure this method is necessary, you can just cast the derived pointer directly